chimedb/core/connectdb.py

```diff
@@ -262,7 +237,7 @@ def execute_sql(self, sql, params=None, commit=True):
         return cursor
 
 
-class MySQLDatabaseReconnect(RetryOperationalError, pw.MySQLDatabase):
+class MySQLDatabaseReconnect(RetryOperationalError, MySQLConnectorDatabase):
```

Comment:
What's up with this?

Reply:
I'm not following. What's wrong with it?

Reply:
Sorry.  Why are you changing it?  Was it a Py2 thing?

Reply:
No. It's switching peewee's backend to the pure python connector (i.e. the actual point of this PR).

Reply:
ahhhhhhhhhhhhhhhhhh